### PR TITLE
Styles for edit and views

### PR DESCRIPTION
**Pull request by DevLuis123**
Styles for Edit user and another changes for views

### CODE REVIEW

File: app/views/devise/registrations/new.html.erb [hunk 1]
```diff
@@ -8,6 +8,11 @@
 
       <%= render "devise/shared/error_messages", resource: resource%>
 
+      <div>
+        <%= f.label :username, class: 'font-bold text-x'%><br />
```
Comment (juperameza) [inline app/views/devise/registrations/new.html.erb]: Don't use br

File: app/views/events/edit.html.erb [hunk 2]
```diff
@@ -1,2 +1,4 @@
-<h1>Edit event</h1>
-<%= render 'form', event: @event %>
+<div class="p-10">
+    <h1>Edit event</h1>
+    <%= render 'form', event: @event %>
+</div>
```
Comment (juperameza) [inline app/views/events/edit.html.erb]: Add a new line at the end of this file

File: app/views/events/index.html.erb [hunk 3]
```diff
@@ -1,10 +1,12 @@
...
-<% end %>
+  <% @events.each_with_index do |event, index| %>    
+    <h2><%= "Event no. #{format("%02d", (index + 1).to_s)} - #{event.title.capitalize}" %></h2>
+    <%= preview_event_image event if event.image_is_saved_and_exists? %>
+    <div><%= link_to 'Show details', event_path(event), class: 'btn-2 no-underline' %></div>
+  <% end %>
 
-<div class="mt-6"><%== pagy_nav(@pagy) if @pagy.pages > 1 %></div>
+  <div class="mt-6"><%== pagy_nav(@pagy) if @pagy.pages > 1 %></div>
+</div>
```
Comment (juperameza) [inline app/views/events/index.html.erb]: Same here new line

File: app/views/events/show.html.erb [hunk 4]
```diff
@@ -1,11 +1,13 @@
...
+    <%= render 'event', event: @event %>
+    <div class="flex">
+      <div class="mb-3"><%= link_to 'Edit event', edit_event_path(@event), class: 'btn-1 py-1 no-underline mb-6' %></div>
+      
+      <%= button_to 'Delete event', event_path(@event), class: 'btn-2 mb-3 ml-3', 
+      method: :delete, data: {turbo_confirm: "Are you sure?"} %> 
+      
+      <div class="mb-3"><%= link_to 'Back', events_path, class: 'btn-3 py-1 ml-3 no-underline' %></div>
+  </div>
+</div>
```
Comment (juperameza) [inline app/views/events/show.html.erb]: Same here new line

File: db/migrate/20240201003641_add_username_to_users.rb [hunk 5]
```diff
@@ -2,6 +2,7 @@
 
 class AddUsernameToUsers < ActiveRecord::Migration[7.1]
   def change
-    add_column :users, :username, :string, unique: true
+    add_column :users, :username, :string
+    add_index :users, :username, unique: true
   end
-end
+end
```
Comment (juperameza) [inline db/migrate/20240201003641_add_username_to_users.rb]: Same here new line

**Review by juperameza — CHANGES_REQUESTED** [review]

File: app/views/devise/confirmations/new.html.erb [hunk 6]
```diff
@@ -3,7 +3,7 @@
 <%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f| %>
   <%= render "devise/shared/error_messages", resource: resource %>
 
-  <div class="field">
+  <div class="">
```
Comment (MauricioNC) [inline app/views/devise/confirmations/new.html.erb]: If you don't need to add classes, delete the class attribute

File: app/views/events/edit.html.erb (hunk 2 above)
Comment (MauricioNC) [inline app/views/events/edit.html.erb]: Add an empty line at the end of your code

File: app/views/events/index.html.erb (hunk 3 above)
Comment (MauricioNC) [inline app/views/events/index.html.erb]: Same here.
Add an empty line at the end of your code

File: app/views/events/show.html.erb (hunk 4 above)
Comment (MauricioNC) [inline app/views/events/show.html.erb]: Same here.
Add an empty line at the end of your code

File: db/migrate/20240201003641_add_username_to_users.rb (hunk 5 above)
Comment (MauricioNC) [inline db/migrate/20240201003641_add_username_to_users.rb]: Same here.
Add an empty line at the end of your code

**Review by MauricioNC — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/application.tailwind.css [hunk 7]
```diff
@@ -12,6 +12,9 @@
   .btn-3{
     @apply bg-red-400 hover:bg-red-500 text-black font-bold px-4 rounded-xl;
   }
+  .btn_4{
```
Comment (JuanequeX) [inline app/assets/stylesheets/application.tailwind.css]: Add a space before your curly brace

File: app/views/devise/registrations/edit.html.erb [hunk 8]
```diff
@@ -1,43 +1,61 @@
...
+        <div>Currently waiting confirmation for: <%= resource.unconfirmed_email %></div>
+      <% end %>
+
+      <div>
+        <%= f.label :password, class: 'font-bold text-x' %> <i>(leave blank if you don't want to change it)</i><br />
+      </div>
+      <div>
+        <%= f.password_field :password, autocomplete: "new-password", class: 'rounded mb-2'  %>
+        <% if @minimum_password_length %>
+        <% end %>
```
Comment (JuanequeX) [inline app/views/devise/registrations/edit.html.erb]: I think that this condition doesn't have any purpose, or what is the reason for it?

File: app/views/devise/registrations/edit.html.erb [hunk 9]
```diff
@@ -1,43 +1,61 @@
...
+      </div>
+
+      <div>
+        <%= f.password_field :current_password, autocomplete: "current-password", class: 'rounded mb-2'  %>
+      </div>
+
+      <div>
+      <%= button_to "Cancel my account?", 
+      registration_path(resource_name), data: { confirm: "Are you sure?", 
+      turbo_confirm: "Are you sure?" }, method: :delete, class: 'underline mb-4 hover:text-red-600 ' %>
```
Comment (JuanequeX) [inline app/views/devise/registrations/edit.html.erb]: indented correctly this button

File: app/views/events/edit.html.erb (hunk 2 above)
Comment (JuanequeX) [inline app/views/events/edit.html.erb]: Indentend correctly this file and add a line at the end of the div.

File: app/views/events/index.html.erb (hunk 3 above)
Comment (JuanequeX) [inline app/views/events/index.html.erb]: same here with the last line at the end of the file

File: app/views/events/new.html.erb [hunk 10]
```diff
@@ -1,2 +1,4 @@
-<h1>New event</h1>
-<div><%= render 'form', event: @event %></div>
+<div class="p-10">
+    <h1>New event</h1>
+    <div><%= render 'form', event: @event %></div>
+</div>
```
Comment (JuanequeX) [inline app/views/events/new.html.erb]: indented correctly, please

**Review by JuanequeX — DISMISSED** [review]

File: app/views/events/edit.html.erb (hunk 2 above)
Comment (DevLuis123) [reply app/views/events/edit.html.erb]: fixed!!

File: app/views/devise/registrations/new.html.erb (hunk 1 above)
Comment (DevLuis123) [reply app/views/devise/registrations/new.html.erb]: fixed!!

File: db/migrate/20240201003641_add_username_to_users.rb (hunk 5 above)
Comment (DevLuis123) [reply db/migrate/20240201003641_add_username_to_users.rb]: fixed!!

File: app/views/devise/confirmations/new.html.erb (hunk 6 above)
Comment (DevLuis123) [reply app/views/devise/confirmations/new.html.erb]: fixed!!

File: app/views/events/edit.html.erb (hunk 2 above)
Comment (DevLuis123) [reply app/views/events/edit.html.erb]: fixed!!

File: app/assets/stylesheets/application.tailwind.css (hunk 7 above)
Comment (DevLuis123) [reply app/assets/stylesheets/application.tailwind.css]: fixed!!

File: app/views/devise/registrations/edit.html.erb (hunk 9 above)
Comment (DevLuis123) [reply app/views/devise/registrations/edit.html.erb]: fixed!!

File: app/views/events/index.html.erb (hunk 3 above)
Comment (DevLuis123) [reply app/views/events/index.html.erb]: fixed!!

File: app/views/events/new.html.erb (hunk 10 above)
Comment (DevLuis123) [reply app/views/events/new.html.erb]: fixed!!

File: app/views/events/edit.html.erb (hunk 2 above)
Comment (DevLuis123) [reply app/views/events/edit.html.erb]: fixed!!

File: app/views/devise/registrations/edit.html.erb (hunk 8 above)
Comment (DevLuis123) [reply app/views/devise/registrations/edit.html.erb]: fixed!!

File: app/views/events/index.html.erb (hunk 3 above)
Comment (DevLuis123) [reply app/views/events/index.html.erb]: fixed!!

File: app/views/events/show.html.erb (hunk 4 above)
Comment (DevLuis123) [reply app/views/events/show.html.erb]: fixed!!

File: app/views/events/index.html.erb (hunk 3 above)
Comment (DevLuis123) [reply app/views/events/index.html.erb]: fixed!!

**Review by MauricioNC — APPROVED** [review]

**Review by juperameza — APPROVED** [review]
Lgtm

**Review by MauricioNC — APPROVED** [review]

**Pull request merged**